app/presenters/publishing_api/case_study_presenter.rb

```diff
@@ -24,6 +24,7 @@ def content
           public_updated_at: item.public_timestamp || item.updated_at,
           rendering_app: Whitehall::RenderingApp::GOVERNMENT_FRONTEND,
           schema_name: "case_study",
+          auth_bypass_ids: [item.auth_bypass_id],
```

Comment:
@baisa, just a thought based off our chat last week. Code changes like this will cause problems until we've backfilled all the past editions with auth_bypass_ids. 

You might need to break this up into a few PRs:

1) create auth_bypass_ids on new editions
2) backfill old editions with auth_bypass_ids
3) set auth_bypass_ids to non null
4) send them to Publishing API

Reply:
Hey Kevin, I can't see anywhere in publishing api that suggests it doesn't accept arrays with nil, and e.g. this [test data](https://github.com/alphagov/publishing-api/blob/dd8dd9232d3cbf33b8945fdd898ebe80d7dcfcf6/spec/support/request_helpers/mocks.rb#L53) suggests that it can. (Also just wrote a Command test to see, but don't know what it's mocking out).

Am I missing something here? (Imagine this is v probable :-) ) Original thinking was that it'd be nice to be able to deploy a fully end to end thing, but more than happy to split it out (and obviously need to do it if it's not safe).